components/mpas-framework/src/framework/mpas_moabmesh.F

```diff
@@ -0,0 +1,205 @@
+module mpas_moabmesh
+!  use, intrinsic :: ISO_C_BINDING
```

Comment:
There seems to be a lot of commented-out code here that should be removed before this gets merged.